Source/Scene/Model.js

```diff
@@ -2972,11 +3069,58 @@ define([
                     castShadows : castShadows,
                     receiveShadows : receiveShadows,
                     uniformMap : uniformMap,
-                    renderState : rs,
+                    renderState : drawRS,
                     owner : owner,
                     pass : isTranslucent ? Pass.TRANSLUCENT : Pass.OPAQUE
                 });
 
+                // Setup the stencil command for the highlight
+                var highlightRS = clone(rs);
+                highlightRS.stencilTest = {
```

Comment:
I agree, we can look at modifying the shader later to discard out parts with alpha = zero, but perhaps better to wait until it's really necessary and we have a model that really exhibits the problem.

I originally had this culling out front faces like you mentioned, which is why I had the cull face set to WebGLConstants.FRONT.  It works, but didn't look as nice on most of the test models as just disabling cull face altogether.  You start to get gaps and breaks in the outline really fast as you increase the size.

I believe the issue is that the shell expansion techniques are assuming that you are dealing with a continuous closed mesh that will extrude as a whole, instead of dealing with individual nodes in the models that are disconnected and extrude as separate parts.